osidb/models.py

```diff
@@ -1138,6 +1157,33 @@ def bzsync(self, *args, bz_api_key, **kwargs):
         fc = FlawCollector()
         fc.sync_flaw(self.bz_id)
 
+    def tasksync(self, *args, jira_token, **kwargs):
+        """
+        Task sync of the Flaw instance in Jira
+
+        This function is responsible for synchronizing only existing flaws
+        that were authored in OSIDB which means they have a task already
+        created in Jira.
+        """
+        # imports here to prevent cycles
+        from apps.taskman.service import JiraTaskmanQuerier
+
+        # new instances are sync-ed in FlawManager.create
```

Comment:
I actually think this method should handle new instances too, then you decouple the jira token check from the part that handles the actual creation/update of tasks, and it allows us to easily trigger task creation/update manually. Imagine that for x reason a JIRA task was not created during flaw creation, we'd have to recreate the flaw to create the JIRA task... (in practice we'd probably call `create_or_update_task` manually, but you get my point).